common/src/main/java/com/hibiscusmc/hmccosmetics/user/CosmeticUser.java

```diff
@@ -77,10 +78,19 @@ public Cosmetic getCosmetic(CosmeticSlot slot) {
         return playerCosmetics.get(slot);
     }
 
+    @Deprecated
     public Collection<Cosmetic> getCosmetic() {
         return playerCosmetics.values();
     }
 
+    public Collection<Cosmetic> getCosmetics() {
```

Comment:
I am a bit confused why we would be making such a change if we don't also make this Immutable. We just deprecated getCosmetic() for no reason then.

Reply:
You’re right I’ll apply the change in a bit